etc/profile-a-l/celluloid.profile

```diff
@@ -9,6 +9,7 @@ include globals.local
 noblacklist ${HOME}/.config/celluloid
 noblacklist ${HOME}/.config/gnome-mpv
 noblacklist ${HOME}/.config/youtube-dl
+noblacklist /usr/lib/liblua*
```

Comment:
I think for allowing Lua it's better to include allow-lua.inc (which also has this line, but can be easily changed in the future).

Reply:
done. didn't notice `allow-lua.inc` at first, otherwise would have included that.